src/neo/Ledger/StorageItem.cs

```diff
@@ -9,6 +10,18 @@ namespace Neo.Ledger
 {
     public class StorageItem : ICloneable<StorageItem>, ISerializable
     {
+        private static readonly uint MaxStackSize;
+        private static readonly uint MaxItemSize;
+
+        static StorageItem()
+        {
+            // Init default values with the the default application engine values
+
+            using var engine = ApplicationEngine.Create(TriggerType.All, null, null);
```

Comment:
It looks weird.

Reply:
We have two other options: send the `ApplicationEngine` to `Get Interoperable` as an argument or hardcode the same values as constants.

Reply:
Please take a look to https://github.com/neo-project/neo-vm/pull/373

Reply:
Good!

Reply:
Done